src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AbstractRazorDelegatingEndpoint.cs

```diff
@@ -159,6 +172,7 @@ protected virtual Task<TResponse> HandleDelegatedResponseAsync(TResponse delegat
         TResponse? delegatedRequest;
         try
         {
+            using var _ = _telemetryReporter?.TrackLspRequest(nameof(HandleRequestAsync), LspName, positionInfo.LanguageKind == RazorLanguageKind.CSharp ? "Razor C# Language Server Client" : "HtmlDelegationLanguageServerClient", correlationId);
```

Comment:
If we're planning to use the same telemetry name for all requests (which I think is a good idea), we should just remove the first argument here.